install/v0.1.0/kfserving.yaml

```diff
@@ -0,0 +1,633 @@
+  namespace: kfserving-system
+---
+apiVersion: v1
+data:
+  credentials: |-
+    {
+       "gcs" : {
+           "gcsCredentialFileName": "gcloud-application-credentials.json"
+       },
+       "s3" : {
+           "s3AccessKeyIDName": "awsAccessKeyID",
+           "s3SecretAccessKeyName": "awsSecretAccessKey"
+       }
+    }
+  frameworks: |-
+    {
+        "tensorflow" : {
+            "image" : "tensorflow/serving"
+        },
+        "sklearn" : {
+            "image" : "gcr.io/kfserving/sklearnserver"
+        },
+        "xgboost" : {
+            "image" : "gcr.io/kfserving/xgbserver"
+        },
+        "tensorrt" : {
+            "image": "nvcr.io/nvidia/tensorrtserver"
+        }
+    }
+kind: ConfigMap
+metadata:
+  name: kfservice-config
+  namespace: kfserving-system
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kfserving-webhook-server-secret
+  namespace: kfserving-system
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/port: "8443"
+    prometheus.io/scheme: https
+    prometheus.io/scrape: "true"
+  labels:
+    control-plane: controller-manager
+    controller-tools.k8s.io: "1.0"
+  name: kfserving-controller-manager-metrics-service
+  namespace: kfserving-system
+spec:
+  ports:
+  - name: https
+    port: 8443
+    targetPort: https
+  selector:
+    control-plane: controller-manager
+    controller-tools.k8s.io: "1.0"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    control-plane: kfserving-controller-manager
+    controller-tools.k8s.io: "1.0"
+  name: kfserving-controller-manager-service
+  namespace: kfserving-system
+spec:
+  ports:
+  - port: 443
+  selector:
+    control-plane: kfserving-controller-manager
+    controller-tools.k8s.io: "1.0"
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    control-plane: kfserving-controller-manager
+    controller-tools.k8s.io: "1.0"
+  name: kfserving-controller-manager
+  namespace: kfserving-system
+spec:
+  selector:
+    matchLabels:
+      control-plane: kfserving-controller-manager
+      controller-tools.k8s.io: "1.0"
+  serviceName: controller-manager-service
+  template:
+    metadata:
+      labels:
+        control-plane: kfserving-controller-manager
+        controller-tools.k8s.io: "1.0"
+    spec:
+      containers:
+      - args:
+        - --secure-listen-address=0.0.0.0:8443
+        - --upstream=http://127.0.0.1:8080/
+        - --logtostderr=true
+        - --v=10
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.4.0
+        name: kube-rbac-proxy
+        ports:
+        - containerPort: 8443
+          name: https
+      - args:
+        - --metrics-addr=127.0.0.1:8080
+        command:
+        - /manager
+        env:
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: SECRET_NAME
+          value: kfserving-webhook-server-secret
+        image: gcr.io/kfserving/kfserving-controller:latest
```

Comment:
I think we'd better rename `gcr.io/kfserving/kfserving-controller:latest` to `gcr.io/kfserving/kfserving-controller:v0.1.0`

Reply:
Infact we should, given this a release, we should not be pulling from master

Reply:
Wow this completely skipped my mind. Thank you!